src/components/common/PageFramework/PageFramework.tsx

```diff
@@ -1,41 +1,60 @@
-import { PropsWithChildren, useEffect } from 'react'
+import React, { PropsWithChildren } from 'react'
 import { useIntl } from 'react-intl'
 import { useLocation } from 'react-router-dom'
 
 import { Heading } from '@navikt/ds-react'
 import clsx from 'clsx'
 
+import KalkulatorLogo from '../../../assets/kalkulator.svg'
+
 import styles from './PageFramework.module.scss'
 
 export const PageFramework: React.FC<
```

Comment:
Denne bør kanskje få utvidet tester for å dekke disse props mer skikkelig. Men kan komme som tillegg til denne PR etterhvert.